### PR TITLE
make `/status` RPC endpoint resistant to consensus halt

### PR DESCRIPTION
**Pull request by melekes**
Refs #1772 

validator power is `0` when consensus halts

don't think we should do the same for /dump_consensus_state or /consensus_state endpoints cause their entire output depends on consensus state, which we can't get due to mutex.

* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGELOG.md

### CODE REVIEW

**Comment by melekes** [comment]
> Does it result in an error for the caller of the endpoint?

no, just 0 validator power when consensus halts

File: rpc/core/status.go [hunk 1]
```diff
@@ -4,12 +4,14 @@ import (
...
 	"time"
 
+	cmn "github.com/tendermint/tendermint/libs/common"
 	ctypes "github.com/tendermint/tendermint/rpc/core/types"
 	sm "github.com/tendermint/tendermint/state"
 	"github.com/tendermint/tendermint/types"
-	cmn "github.com/tendermint/tendermint/libs/common"
 )
 
+const consensusTimeout = time.Second
```
Comment (melekes) [inline rpc/core/status.go]: I'd prefer this const be inside the method (closer to the usage). Is there a specific reason to keep it in the top of the file?

Comment (xla) [reply rpc/core/status.go]: Gonna move.

Comment (xla) [reply rpc/core/status.go]: But I don't see much valuein having the constants inside of methods.

Comment (melekes) [reply rpc/core/status.go]: developer does not have to jump to definition in order to see the value

**Comment by jaekwon** [comment]
This is addressing the symptom and ignoring the underlying cause of the problem.  Please revert asap.

**Pull request merged**